src/jupiterone/entities/UserEntity.ts

```diff
@@ -0,0 +1,9 @@
+import { EntityFromIntegration } from "@jupiterone/jupiter-managed-integration-sdk";
+
+export const USER_ENTITY_TYPE = "jamf_user";
+export const USER_ENTITY_CLASS = "Person";
```

Comment:
The `Person` class is reserved for use in "global" entities that are created automatically by mapping `User` entities having an `email` property.

```suggestion
export const USER_ENTITY_CLASS = "User";
```